test/expect-tests/dune_console/dune_console_tests.ml

```diff
@@ -1,14 +1,17 @@
-(** Creation of Dune_console is stateful so we introduce a new module for each test. *)
+open Stdune
+
+let escape str =
+  str |> String.split_lines |> List.map ~f:String.escaped |> List.iter ~f:prerr_endline
```

Comment:
Any specific reason why we're printing to stderr?

Reply:
No. Shall I switch to stdout?

Reply:
Yes